dags/merino_jobs.py

```diff
@@ -171,6 +174,9 @@ def merino_job(
         subject="Navigational Suggestions Domain Metadata job successful",
         html_content="""
```

Comment:
```suggestion
        html_content=f"""
```

Reply:
i committed and then reverted because i remembered that this is a jinja template and not a fstring :)